src/auth/helper/auth.helper.ts

```
@@ -1,30 +1,63 @@
+
+    async creteUser(dto: signUpInputDto, hash: string) {
+        try {
+            const user = await this.prisma.user.create({
+                data: {
+                    email: dto.email,
+                    password: hash,
+                    firstName: dto.firstName,
+                    lastName: dto.lastName
+                },
+            });
+            delete user.password;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)